### PR TITLE
versions: Linux 5.15.x

### PR DESCRIPTION
**Pull request by jcvenegas**
Upgrade to new Linux kernel LTS version.

Fixes: #3576

Signed-off-by: Carlos Venegas <jose.carlos.venegas.munoz@intel.com>

### CODE REVIEW

**Review by fidencio — APPROVED** [review]
lgtm, thanks @jcvenegas!

File: versions.yaml [hunk 1]
```diff
@@ -155,7 +155,7 @@ assets:
   kernel:
     description: "Linux kernel optimised for virtual machines"
     url: "https://cdn.kernel.org/pub/linux/kernel/v5.x/"
-    version: "v5.10.25"
+    version: "v5.15.17"
```
Comment (gkurz) [inline versions.yaml:158]: According to https://www.kernel.org/ , the LTS version was bumped to .18 two days ago:
```
longterm:     5.15.18     2022-01-29
```
Maybe worth doing the same in kata ?

**Review by gkurz — CHANGES_REQUESTED** [review]

File: versions.yaml (hunk 1 above)
Comment (jcvenegas) [reply versions.yaml]: Updated to `.19` :)

**Review by fidencio — APPROVED** [review]
lgtm, and addresses @gkurz's comments.

**Review by gkurz — APPROVED** [review]

File: tools/packaging/kernel/configs/fragments/whitelist.conf [hunk 2]
```diff
@@ -12,3 +12,6 @@ CONFIG_CRYPTO_DEV_SP_PSP
 CONFIG_CRYPTO_DEV_CCP
 CONFIG_HAVE_NET_DSA
 CONFIG_NF_LOG_COMMON
+CONFIG_MANDATORY_FILE_LOCKING
+CONFIG_ARM64_UAO
```
Comment (bergwolf) [inline tools/packaging/kernel/configs/fragments/whitelist.conf:16]: Should it belong to somewhere ARM specific?

Comment (fidencio) [reply tools/packaging/kernel/configs/fragments/whitelist.conf:16]: @bergwolf, you're totally right!

**Comment by fidencio** [comment]
Superseded by https://github.com/kata-containers/kata-containers/pull/3684

**Pull request closed without merging**